include/swift/AST/DiagnosticsSema.def

```diff
@@ -704,7 +704,7 @@ ERROR(sema_no_import,Fatal,
       "no such module '%0'", (StringRef))
 ERROR(sema_no_import_target,Fatal,
       "could not find module '%0' for target '%1'; "
-      "found: %2", (StringRef, StringRef, StringRef))
+      "found: %2, at:\n%3", (StringRef, StringRef, StringRef, StringRef))
```

Comment:
Does it need a new line character here? I'm not used to seeing those in our diagnostic.

Reply:
You're right, other diagnostics don't use newlines. Fixed.